prog/vm/nexus.rb

```diff
@@ -457,14 +457,15 @@ def write_params_json
   end
 
   label def wait_lb_expiry
-    unless vm.lb_expiry_started_set?
-      vm.incr_lb_expiry_started
-      vm.load_balancer.evacuate_vm(vm)
-      nap 30
+    if vm.load_balancer
```

Comment:
maybe: `if (lb = vm.load_balancer)` and then replace `vm.load_balancer` with `lb` inside `if`.